<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/images.qrc">
    <normaloff>:/icon/images/3dPrinter.png</normaloff>:/icon/images/3dPrinter.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/images.qrc">
      <normaloff>:/icon/images/tools.png</normaloff>:/icon/images/tools.png</iconset>
    </property>
    <addaction name="_network_action"/>
    <addaction name="_refresh_devices_action"/>
    <addaction name="separator"/>
    <addaction name="_task_manager_action"/>
    <addaction name="_show_tasks_window_action"/>
   </widget>
   <widget class="QMenu" name="menucamera">
    <property name="title">
     <string>camera</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/images.qrc">
      <normaloff>:/icon/images/camera.png</normaloff>:/icon/images/camera.png</iconset>
    </property>
    <addaction name="actioncapture"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menucamera"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="_network_action">
   <property name="icon">
    <iconset resource="../resources/images.qrc">
     <normaloff>:/icon/images/tools.png</normaloff>:/icon/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>network info</string>
   </property>
  </action>
  <action name="_refresh_devices_action">
   <property name="icon">
    <iconset resource="../resources/images.qrc">
     <normaloff>:/icon/images/tools.png</normaloff>:/icon/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>refresh devices</string>
   </property>
  </action>
  <action name="_task_manager_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>enable task manager</string>
   </property>
  </action>
  <action name="_show_tasks_window_action">
   <property name="icon">
    <iconset resource="../resources/images.qrc">
     <normaloff>:/icon/images/tasks.png</normaloff>:/icon/images/tasks.png</iconset>
   </property>
   <property name="text">
    <string>show tasks window</string>
   </property>
  </action>
  <action name="actioncapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>enable camera captures</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
